ReadMe.md

```diff
@@ -1,56 +1,55 @@
-OP2MapImager - Outpost 2 Map and Saved Game Imager
-Developed by Hooman and Brett208 (Vagabond)
+# OP2MapImager
 
-Check the OP2MapImager landing page for the newest version at https://wiki.outpost2.net/doku.php?id=outpost_2:helper_programs:op2mapimager.
+Outpost 2 Map and Saved Game Imager utility. This utility allows rendering a map or saved game file (.map, .op2) into an image file (.png, .jpg, .bmp). It supports individual map imaging as well as batch processing of all maps in a folder. Check the [OP2MapImager landing page](https://wiki.outpost2.net/doku.php?id=outpost_2:helper_programs:op2mapimager) for the newest version. Developed by Hooman and Brett208 (Vagabond).
 
-+++ USAGE NOTES +++
+## USAGE NOTES
   * Capable of saving multiple map files and entire directories.
   * The OP2MapImager executable and FreeImage.dll must be in the same directory as the tileset BMPs.
   * Scale Factor (-s) determines the final render size and represents the final pixel length of a single tile
-    * Max Value: 32, renders at full size, or 32 pixels per tile.
-    * Min Value: 1, renders at 1 pixel per tile
-
-+++ EXAMPLE COMMANDS +++
-  * OP2MapImager mapFilename.[map|OP2]
-  * OP2MapImager -s 16 -o -q Ashes.map eden01.map sgame0.op2
-  * OP2MapImager --Scale 8 --ImageFormat BMP [Directory of choice]
-
-+++ OPTIONAL ARGUMENTS +++
-  -H / --Help: Displays Help File
-  -Q / --Quiet: [Default false] Add switch to run application without issuing console messages.
-  -O / --Overwrite: [Default false] Add switch to allow application to overwrite existing files.
-  -D / --DestinationDirectory: [Default MapRenders]. Add switch and name of new destination path.
-  -I / --ImageFormat: [Default PNG]. Allows PNG|JPG|BMP. Sets the image format of the final render.
-  -S / --Scale: [Default 4] Sets Scale Factor of image.
-  -A / --AccessArchives [Default true]. Add switch to disable searching VOL archives for map and well files.
+    * Min Value: 1, renders at 1 pixel per tile (minimap view)
+    * Max Value: 32, renders at 32 pixels per tile (full size detail pane view)
```

Comment:
I think this would read better as full size map instead of full size detail pane view.

Reply:
I used "detail pane" because that's what Outpost 2 calls it in the documentation. If I read "full size map", I'm not sure I would understand what they were trying to convey.

Reply:
I think I'm going to go with your suggestion on this one.